src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/configs/decider/jvm/LevelOneActionBuilderConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs.decider.jvm;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs.ConfigUtils;
+import java.util.Map;
+
+public class LevelOneActionBuilderConfig {
+  public static final double DEFAULT_FIELD_DATA_CACHE_LOWER_BOUND = 0.1;
+  public static final double DEFAULT_SHARD_REQUEST_CACHE_LOWER_BOUND = 0.02;
+  private static String FIELD_DATA_CACHE_LOWER_BOUND_CONFIG_NAME = "fielddata-cache-lower-bound";
+  private static String SHARD_REQUEST_CACHE_LOWER_BOUND_CONFIG_NAME = "shard-request-cache-lower-bound";
```

Comment:
Let's move these to `CacheActionConfig`? we can call them `soft-upper/lower-bound`.